lms/djangoapps/course_structure_api/v0/tests.py

```diff
@@ -208,9 +208,14 @@ def test_not_authorized(self):
         self.assertEqual(response.status_code, 404)
 
 
-class CourseListTests(CourseViewTestsMixin, ModuleStoreTestCase):
+class CourseListTests(CourseViewTestsMixin, SharedModuleStoreTestCase):
     view = 'course_structure_api:v0:list'
 
+    @classmethod
+    def setUpClass(cls):
```

Comment:
Right, I think it's bad practice in general, but in this particular case (`CourseViewTestsMixin`) there are only two stub tests that don't do anything (that I could just delete). But anyhow, just having a little redundancy in the sub-classes is probably better in the long term.